src/main/java/org/eclipse/xpanse/terraform/boot/models/exceptions/TerraformApiExceptionHandler.java

```diff
@@ -140,4 +140,28 @@ public Response handleInvalidTerraformScriptsException(
         return Response.errorResponse(ResultType.INVALID_TERRAFORM_SCRIPTS,
                 Collections.singletonList(ex.getMessage()));
     }
+
+    /**
+     * Exception handler for FailedCallbackResponseStoreException.
+     */
+    @ExceptionHandler({FailedCallbackResponseStoreException.class})
```

Comment:
do we really need an ExceptionHandler for this? This exception is not thrown as part of any API calls. We must only log this exception and exit.